### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by KanbH**
-Added a Register page.
-Created the Register page.
-Integrated the Register page into the router.
-Made changes to the InputForm component.

### CODE REVIEW

**Review by PhupaSirirat — CHANGES_REQUESTED** [review]
Dont forget to 'git merge main'. And resolve conflict.

**Comment by KanbH** [comment]
Conflict resolved

File: src/Register.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      <h1 className={headerText()}>ลงทะเบียน</h1>
+      <section className="w-4/5">
+        <form className="flex flex-col">
+          <InputForm
+            label="หมายเลขโทรศัพท์/อีเมล์"
+            id="username"
+            placeholder="name@email.com"
+          />
+          <InputForm label="รหัสผ่าน" type="password" id="password" hint="รหัสผ่านต้องมีความยาวอย่างน้อย 8 อักษร"/>
+          <InputForm label="ยืนยันรหัสผ่าน" type="confirmpassword" id="confirmpassword" />
```
Comment (PhupaSirirat) [inline src/Register.jsx]: This type of InputFrom should be 'password'.
```suggestion
          <InputForm label="ยืนยันรหัสผ่าน" type="password" id="confirmpassword" />
```

File: src/components/InputForm.jsx [hunk 2]
```diff
@@ -26,6 +26,9 @@ export function InputForm({label, type, id, placeholder}) {
           placeholder={placeholder}
           required
         />
+        <label className="block mb-2 text-xs font-medium text-gray-900 ">
+          {hint}
+        </label>
```
Comment (PhupaSirirat) [inline src/components/InputForm.jsx:31]: We already have label at the top of this code.

**Review by PhupaSirirat — CHANGES_REQUESTED** [review]
Sorry, forgot to submit review. lol

**Review by PhupaSirirat — APPROVED** [review]

**Pull request merged**